lib/wicked/wlan.pm

```diff
@@ -413,7 +442,8 @@ sub run {
         for my $ifcfg_wlan (__as_config_array($self->ifcfg_wlan())) {
             $self->hostapd_start($hostapd_conf->{config});
 
-            if ($self->check_wicked_version($ifcfg_wlan->{wicked_version})) {
+            if ($self->check_wicked_version($ifcfg_wlan->{wicked_version}) &&
```

Comment:
and why we need to do same check two times ?

Reply:
The first one is for "test-global" version specification. The second one for each config.

As example:
If a full tests can be skipped duo to version check: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/wicked/wlan/sut/t07_multi_networks_wpa_psk.pm#L23
or only a single config out of a test: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/wicked/wlan/sut/t03_wpa_psk.pm#L94

Reply:
nice !